cli/clone/register.ts

```diff
@@ -75,7 +75,9 @@ export const registerClone = (program: Command) => {
             })
             .json()
 
-          const fullPath = path.join(dirPath, filePath)
+          let fullPath = path.join(dirPath, filePath)
+          if (filePath === "index.ts")
+            fullPath = fullPath.replace(".ts", ".tsx")
```

Comment:
we can't modify the files from the cloud (think of how many problems this would cause with version control), this issue is with the registry api storing package content in `index.ts` instead of `index.tsx` CC @imrishabh18